tests/python-tests/CWE-502/local/unsafe.py

```diff
@@ -0,0 +1,17 @@
+import os
+import pickle
+
+# > Local input
+i = input()
+
+pickle.loads(i)
+
+# > Local Environment Variables
+e = pickle.dumps(os.environ.get('LOCAL_DATA'))
+
+pickle.loads(e)
```

Comment:
There is no corresponding output within [tests/python-tests/CWE-502/local/UnsafeDeserializationLocal.expected](https://github.com/advanced-security/codeql-queries/pull/48/files#diff-5d00d9eadc1c0289e5292252d9d5586d0af39811faae5f454deaabbf8bb60a8e) for this line. Is that intended? If so, perhaps it would make sense to put a comment here, saying that we can't yet detect it, e.g. `known false positive:`

Reply:
Absolutely right. Patch coming soon

Reply:
Absolutely right. Patch coming soon

Reply:
Fixed with latest commit